src/Lykke.AlgoStore.CSharp.AlgoTemplate.Models/Mapper/AlgoRatingsMapper.cs

```diff
@@ -56,6 +78,20 @@ public static AlgoRatingEntity ToEntity(this AlgoRatingData data)
             return result;
 
         }
+
+        public static AlgoRatingEntity ToClientIdPartionKeyEntity(this AlgoRatingData data)
```

Comment:
Rename method to "ToEntityWithClientIdPartitionKey in order to be consistent with the other such mapper methods

Reply:
done